src/js/components/SearchForm.js

```diff
@@ -0,0 +1,74 @@
+import { entries } from '../util';
+import Option from './Option';
+
+export default ({ }) => {
+  const $search = (
+    <input
+      name="q"
+      id="search"
+      class="gurbani-font"
```

Comment:
className

Reply:
Nice catch, however class will work for our implementation of `h`. It'll also work with `preact` and `react@^16`. So for now we can live with it.